### ISSUE TITLE
Working with archives

### ISSUE THREAD

**Issue by ags1234** [angelbbs/DosNavigator#1]
Hey, thanks for providing binaries for this wonderful software!

1. I have Winrar installed on disk, but how can I work with RAR archives in DN? When I try to enter an archive, nothing happens.
2.  The browsing of zip archives works, but when unpacking, there is an error of pkunzip not found. Winrar can be used to uncompress zip archives in Windows, but how to configure DN to use it?
3.

**Comment by angelbbs** [comment]
Press F10 ->Options->Archives

**Comment by ags1234** [comment]
Yes, I know, I went there and gave the full path to rar.exe, for Rar archives.
Still, DN does not enter rar archives and no decompress dialog at Alt+F2 on archive.
What is missing?

**Comment by ags1234** [comment]
Is there a place where the paths in which DN searches for executables can be configured?

**Comment by angelbbs** [comment]
https://stackoverflow.com/questions/9546324/adding-a-directory-to-the-path-environment-variable-in-windows

**Comment by angelbbs** [comment]
Because it's not Necromancer's Dos Navigator, but DN/2
I do not know who he is AngelsHolocaust

**Comment by ags1234** [comment]
Ok then ;)

Back on the subject, if we find out how to properly configure archives in DN, this thread may help others with the same problem. It should not be closed.

**Comment by angelbbs** [comment]
rar.exe is it launched from the command line? or "not recognized as an internal or external command"?

**Comment by angelbbs** [comment]
my settings
![image](https://github.com/user-attachments/assets/d936c72b-cea0-4a25-93e9-415b495f84c6)

**Comment by ags1234** [comment]
Exactly like mine. I am out of ideas...
